### PR TITLE
[All] Rename Office Online

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
After discussing with our editor, Laura, landed on the following.
-Rename "Office Online" to "Office on the web"
-Reword "connected to Office 365" to "connected to Office 365 subscription"
-Reword from "for [platform]" to "on [platform]"

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[5c2aa2e](https://github.com/OfficeDev/office-js-docs-pr/pull/1062/commits/5c2aa2e233319f4b1c84b604d8bd6a8ce57fe60b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-rebrand/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906071748335366-1062%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=hzYTBslKjzD5DmN%2FXxsiF1lmXs3CzQt4gw7ctSBYnzo%3D&st=2019-06-07T17%3A44%3A37Z&se=2019-07-08T17%3A49%3A37Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[e482508](https://github.com/OfficeDev/office-js-docs-pr/pull/1062/commits/e482508b00e3f910ee296d4f0818a7c39f9da0ec)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-rebrand/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132054493549-1062%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=g%2BX6rq2POhfzf%2F9zzzGXMydtGgI9RsofMusGheBTIlg%3D&st=2019-06-13T20%3A50%3A46Z&se=2019-07-14T20%3A55%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[9f55f79](https://github.com/OfficeDev/office-js-docs-pr/pull/1062/commits/9f55f79337418735e124aaaba968ebf19643ffc3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-rebrand/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132128345219-1062%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=33jgnEgGP%2B%2BN6xTUwImZ8IMe%2F1xp1aF9nUnRkNH5Fc8%3D&st=2019-06-13T21%3A24%3A32Z&se=2019-07-14T21%3A29%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/overview/office-add-in-availability.md [hunk 1]
```diff
@@ -338,7 +338,20 @@ To work as expected, your Office Add-in might depend on a specific Office host,
     <td>Not available</td>
   </tr>
   <tr>
-    <td>Office on Windows<br>(connected to Office 365)</td>
+    <td>Office on the web<br>(Classic)</td>
```
Comment (kbrandl) [inline docs/overview/office-add-in-availability.md]:
```suggestion
    <td>Office on the web<br>(classic)</td>
```

File: docs/overview/office-add-in-availability.md [hunk 2]
```diff
@@ -324,7 +324,7 @@ To work as expected, your Office Add-in might depend on a specific Office host,
     <th><a href="/office/dev/add-ins/reference/requirement-sets/office-add-in-requirement-sets"><b>Common APIs</b></a></th>
   </tr>
   <tr>
-    <td>Office Online</td>
+    <td>Office on the web<br>(Outlook.com or<br>connected to Office 365 subscription)</td>
```
Comment (kbrandl) [inline docs/overview/office-add-in-availability.md]:
```suggestion
    <td>Office on the web<br>(new)</td>
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[df73800](https://github.com/OfficeDev/office-js-docs-pr/pull/1062/commits/df73800756e50b5dfce6d6ece8fcc02abef14fec)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/overview/office-add-in-availability.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-rebrand/docs/overview/office-add-in-availability.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C13%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201906132230173333-1062%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=Sd08Rm7yL%2F5pfdTquC3mNe4uj%2Fl8ErmfaimNViJHQEw%3D&st=2019-06-13T22%3A26%3A17Z&se=2019-07-14T22%3A31%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**